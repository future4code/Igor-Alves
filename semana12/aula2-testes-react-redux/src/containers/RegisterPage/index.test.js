import React from 'react';
import renderer from 'react-test-renderer';
import { RegisterPage } from './index'

it('Snapshot RegisterPage', ()=> {
    const tree = renderer
    .create(<RegisterPage></RegisterPage>)
    .toJSON();
    expect(tree).toMatchSnapshot()
})