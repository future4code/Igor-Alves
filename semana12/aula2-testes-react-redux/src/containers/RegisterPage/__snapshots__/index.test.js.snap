// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot RegisterPage 1`] = `
<div>
  <div
    className="sc-AykKE dYwhLR"
  >
    <header
      className="sc-AykKC dRUEqp"
    >
      <img
        alt="Logo 4eddit"
        className="sc-AykKD eSgZQA"
        src="logoheader.jpg"
      />
    </header>
  </div>
  <form
    className="sc-fzXfNK eJsMNj"
  >
    <h2
      className="sc-fzXfNL gnyjdZ"
    >
      Cadastro
    </h2>
    <div
      className="sc-fzXfNO eGYQTF"
    >
      <label>
        Nome de Usu√°rio
      </label>
      <input
        className="sc-fzXfNM zkwOK"
        name="username"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="sc-fzXfNO eGYQTF"
    >
      <label>
        E-mail
      </label>
      <input
        className="sc-fzXfNM zkwOK"
        name="email"
        onChange={[Function]}
        type="email"
        value=""
      />
    </div>
    <div
      className="sc-fzXfNO eGYQTF"
    >
      <label>
        Senha
      </label>
      <input
        className="sc-fzXfNM zkwOK"
        name="password"
        onChange={[Function]}
        type="password"
        value=""
      />
    </div>
    <button
      className="sc-fzXfNN caCLLX"
      color="primary"
      onClick={[Function]}
      size="large"
      type="submit"
    >
      Cadastrar
    </button>
  </form>
</div>
`;
